app/models/poster.rb

```diff
@@ -1,4 +1,11 @@
 class Poster < ApplicationRecord
+  validates :name, presence: true, uniqueness: true
+  validates :description, presence: true
+  validates :year, presence: true
+  validates :price, presence: true
+  validates :vintage, presence: true
```

Comment:
It seems this validation breaks the functionality when vintage is set to "false", this validation seems to interpret that as the presence is false instead of the value. Here's what chatgpt suggested after I couldn't figure it out

  validate :vintage_presence

  def vintage_presence
    errors.add(:vintage, "can't be blank") if vintage.nil?
  end